pyproject.toml

```diff
@@ -54,6 +54,9 @@ dev = [
     "flake8",                       # Code linter
     "coverage",                     # Test coverage measurement
     "pytest-cov",                   # Test coverage plugin for pytest
+    "statsmodels",                  # Used to compare model pseudo-r2 in testing
+    "scikit-learn",                 # Testing compatibility with CV & pipelines
+    "PyYAML"                          # Load GLM params for testing
```

Comment:
if you change yml to json ,remember to remove this. and the fact that we would only have it be a dev dependency (which makes sense!) makes me more inclined to switch to json, since this will almost certainly trip people up if they try to run the tutorials themselves

Reply:
yep, done it